fishcore/ClientApp/src/components/ExportButton.js

```diff
@@ -0,0 +1,49 @@
+﻿import React, { Component } from 'react';
+
+export class ExportButton extends Component {
```

Comment:
This component adds the export button which reaches out to the server with info about the development window nodes and edges.